controllers/urls.js

```diff
@@ -2,20 +2,21 @@
  * GET /urls
  * URLs page.
  */
-const Url = require('../models/Url')
-const User = require('../models/User')
+const Urls = require('../models/Url')
+const Users = require('../models/User')
```

Comment:
FYI these plural variables broke the other methods in urlControllers, no big deal-- easy fix